src/app/ApplicationTemplate.UWP/Views/Styles/Controls/ListViewItem.xaml

```diff
@@ -156,37 +156,32 @@
 												Value="0.7" />
 									</VisualState.Setters>
 								</VisualState>
-								<VisualState x:Name="Selected">
-									<VisualState.Setters>
-										<Setter Target="CheckIcon.Opacity"
-												Value="1" />
-									</VisualState.Setters>
-								</VisualState>
+								<VisualState x:Name="Selected" />
 							</VisualStateGroup>
 
 						</VisualStateManager.VisualStateGroups>
 
-						<Grid.RowDefinitions>
-							<RowDefinition Height="*" />
-							<RowDefinition Height="Auto" />
-						</Grid.RowDefinitions>
-<!--
+						<Grid.ColumnDefinitions>
+							<ColumnDefinition Width="Auto" />
+							<ColumnDefinition Width="*" />
+						</Grid.ColumnDefinitions>
+
+						<!-- CheckBox -->
+						<CheckBox x:Name="CheckBox"
+								  Content="{TemplateBinding Content}"
+								  Foreground="{TemplateBinding Foreground}"
```

Comment:
No need for IsChecked?

Reply:
It was an error, there's no Binding to Form in the control styles.